docs/geography.rst

```diff
@@ -0,0 +1,85 @@
+Working with Geographic data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+BigQuery provides a `GEOGRAPHY data type
+<https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#geography_type>`_
+for `working with geographic data
+<https://cloud.google.com/bigquery/docs/gis-data>`_, including:
+
+- Points,
+- Linestrings,
+- Polygons, and
+- Collections of points, linestrings, and polygons.
+
+Geographic data uses the `WGS84
+<https://earth-info.nga.mil/#tab_wgs84-data>`_ coordinate system.
+
+To define a geography column, use the `GEOGRAPHY` data type imported
+from the `pybigquery.sqlalchemy_bigquery` module::
+
+  from sqlalchemy.ext.declarative import declarative_base
+  from sqlalchemy import Column, String
+  from pybigquery.sqlalchemy_bigquery import GEOGRAPHY
+
+  Base = declarative_base()
+
+  class Lake(Base):
+      __tablename__ = 'lakes'
+
+      name = Column(String)
+      geog = column(GEOGRAPHY)
```

Comment:
```suggestion
      geog = Column(GEOGRAPHY)
```

Reply:
That's what I get for having docs that aren't automatically tested.  I'll convert this to use a `snippets.py`, following the `python-bigquery` example.